cookbooks/bcpc-hadoop/libraries/utils.rb

```diff
@@ -439,3 +439,22 @@ def update_oozie_sharelib(host)
     Chef::Log.info("Sharelibupdate: Oozie server not running on #{host}")
   end
 end
+
+def get_cluster_nodes()
+
+  cluster_file = node['bcpc']['cluster']['file_path']
+
+  if !File::file?(cluster_file)
+    Chef::Log.fatal("File #{cluster_file} does not exist.")
+    raise
+  end
+
+  fileList = Array.new
+
+  File::open(cluster_file,"r").each_line do |line|
```

Comment:
For reference, it looks like this does what's in #395 [repxe_host.rb](https://github.com/http-418/chef-bach/blob/repxe_script/repxe_host.rb#L209-L220).